### PR TITLE
Updating and adding coffeescript

### PR DESCRIPTION
**Pull request by nmccready**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,21 +15,23 @@
   "author": "Alexey Raspopov (https://github.com/alexeyraspopov)",
   "repository": "alexeyraspopov/gulp-complexity",
   "scripts": {
-    "test": "gulp test"
+    "test": "echo 0"
```
Comment (nmccready) [inline package.json]: if u want this back in np

**Comment by nmccready** [comment]
bump

**Comment by nmccready** [comment]
I can attempt to resolve conflicts; but is there any chance of this being merged? I would like to delete my fork.

**Comment by alexeyraspopov** [comment]
If I'll merge this someone will open the same PR for TypeScript. I don't use these langs and I don't know why this package needs to cover shitty syntactic sugar.

Sorry, I'm closing it.

**Pull request closed without merging**